iroh/src/main.rs

```diff
@@ -52,6 +52,7 @@ async fn main_impl() -> Result<()> {
     #[cfg(feature = "metrics")]
     let metrics_fut = init_metrics_collection(cli.metrics_addr, &rt);
 
+    tracing::debug!(target: "derponly", "{config:?}");
```

Comment:
why print out the full config in this case?

Reply:
🤦‍♀️ I could have sworn i removed that, sigh